Header header
std_msgs/Float32 confidence
geometry_msgs/Point32[] line1
geometry_msgs/Point32[] line2
geometry_msgs/Point32[] line3
std_msgs/String camera_id
